Makefile

```diff
@@ -44,6 +44,9 @@ clean:
 	rm -rf tmp
 	rm -rf *.dat
 
+clean-binaries:
+	rm -f pennylane_lightning/lightning_qubit_ops*
```

Comment:
On linux for me, the wheel gets built in the following directory: `pennylane-lightning/build/lib.linux-x86_64-3.7/pennylane_lightning` and would not be cleaned by this command.

Could we include `python setup.py clean --all` too? Got the following output:
```
/home/antal/anaconda3/lib/python3.7/site-packages/setuptools/dist.py:454: UserWarning: Normalizing '0.11.0-dev' to '0.11.0.dev0'
  warnings.warn(tmpl.format(**locals()))
running clean
removing 'build/temp.linux-x86_64-3.7' (and everything under it)
removing 'build/lib.linux-x86_64-3.7' (and everything under it)
'build/bdist.linux-x86_64' does not exist -- can't clean it
'build/scripts-3.7' does not exist -- can't clean it
removing 'build'
```

Reply:
Actually you're right, now I try again with just `make clean` then it causes me to recompile :thinking: 

Maybe we should just close this PR then?

Reply:
Sounds good